pkg/cmd/server/api/v1/conversions.go

```diff
@@ -43,6 +43,17 @@ func init() {
 				obj.ExecHandlerName = DockerExecHandlerNative
 			}
 		},
+		func(obj *SecurityAllocator) {
+			if len(obj.UIDAllocatorRange) == 0 {
+				obj.UIDAllocatorRange = "1000000000-1999999999/10000"
+			}
+			if len(obj.MCSAllocatorRange) == 0 {
+				obj.MCSAllocatorRange = "s0:/2"
+			}
+			if obj.MCSLabelsPerProject == 0 {
+				obj.MCSLabelsPerProject = 5
```

Comment:
Is this default derivable from the UID and MCS so that it always matches?  Seems like it would be useful for validation as well: "you don't have a enough" or some such.